### PR TITLE
Short names and CRD display name updates

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: api/v1beta1/openstackcontrolplane_types.go [hunk 1]
```diff
@@ -64,6 +64,8 @@ type OpenStackControlPlaneStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=osctlplane;osctlplane
```
Comment (stuggi) [inline api/v1beta1/openstackcontrolplane_types.go:67]: 2nd should be osctlplanes

**Review by stuggi — CHANGES_REQUESTED** [review]

**Comment by abays** [comment]
Other than the item @stuggi pointed out, this looks good

File: api/v1beta1/openstackcontrolplane_types.go (hunk 1 above)
Comment (dprince) [reply api/v1beta1/openstackcontrolplane_types.go:67]: how did I miss this!?

Comment (dprince) [reply api/v1beta1/openstackcontrolplane_types.go]: fixed in latest commit

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**